mautic/mautic
# [UX] Rename Order to Field order and add a tooltip

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The term “Order” currently used in the platform could be more specific. Users might not immediately understand what it refers to. The pull request proposes to rename “Order” to “Field order”, providing more context and reducing ambiguity.

Additionally, adding a tooltip provides an opportunity to give users more information about how these fields are placed practically, without cluttering the interface.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Custom fields
3. Create or edit a field
4. Look on the right side of the screen for "Field order"

![image](https://github.com/mautic/mautic/assets/116097999/87df1005-1817-4633-a544-024a1349a011)
